impl/src/test/java/jakarta/faces/component/UIComponentTestCase.java

```diff
@@ -362,7 +356,7 @@ public void testAttributesTransparency() {
     // Test getChildren().iterator()
     public void testChildrenIterator() {
 
-        Iterator kids;
+		Iterator<UIComponent> kids;
```

Comment:
Reconfigure your IDE to use spaces instead of tabs and fix tabs over all place.

Reply:
Sure. Sorry for that. That was one of my first changes and I forgot to back and correct this.